### ISSUE TITLE
Imitation learning - student not moving in training

### ISSUE THREAD

**Issue by Bakikii** [Unity-Technologies/ml-agents#617]
I've setup my player to follow keyboard actions as required and it has a teacher brain and brain is in player mode. I have a student using same agent script as player.It has a student brain and it's brain is in external mode ..
But there is no movement from student in training. Has anyone any ideas why this is happening or perhaps someone can share a simple working project with imitation learning  . 

Thanks

**Comment by Bakikii** [comment]
Yes, I launch from cmd and it starts up fine,  I can move my teacher agent as expected .... I've followed the git hub instructions to the letter but I'm obviously missing something....

**Comment by vincentpierre** [comment]
Are you able to train the banana imitation environment ? There could be a problem with the connection to python. Does python print summaries as expected or does it timeout after 30 seconds? Make sure you build with `Development Build` checked: this will ensure that any error in unity will be displayed. Also make sure that the student and teacher brain have the same brain parameters (you can just copy and paste the game object to make sure). It might take some time depending on the training_config for the buffer to have enough data points to start training, could take a minute or so depending on the simulation.
